### PR TITLE
1. add new nemesis changing table properties, 2. grow_cluster_test: add new nodes to prometheus

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -365,6 +364,19 @@ def disrupt_nodetool_cleanup(self):
         cmd = 'nodetool -h localhost cleanup keyspace1'
         self._run_nodetool(cmd, self.target_node)
 
+    def disrupt_modify_table_comment(self):
+        self._set_current_disruption('ModifyTableProperties %s' % self.target_node)
```
Comment (amoskong) [inline sdcm/nemesis.py]: `self.target_node.name` is better.

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -365,6 +364,19 @@ def disrupt_nodetool_cleanup(self):
...
+        comment = ''.join(random.choice(string.ascii_letters) for i in xrange(24))
+        cmd = "ALTER TABLE keyspace1.standard1 with comment = '{}';".format(comment)
+        self.target_node.remoter.run('cqlsh -e "{}" {}'.format(cmd, self.target_node.private_ip_address), verbose=True)
+
+    def disrupt_modify_table_gc_grace_sec(self):
+        self._set_current_disruption('ModifyTableProperties %s' % self.target_node)
```
Comment (amoskong) [inline sdcm/nemesis.py]: `self.target_node.name` is better.

File: sdcm/nemesis.py (hunk 1 above)
Comment (larisau) [inline sdcm/nemesis.py]: IMHO string representation of node is more informative - it contains both addresses

File: sdcm/nemesis.py [hunk 3]
```diff
@@ -365,6 +364,19 @@ def disrupt_nodetool_cleanup(self):
...
+        gc_grace_seconds = random.choice(xrange(216000, 864000))
+        cmd = "ALTER TABLE keyspace1.standard1 with comment = 'gc_grace_seconds changed' AND" \
+              " gc_grace_seconds = {};".format(gc_grace_seconds)
+        self.target_node.remoter.run('cqlsh -e "{}" {}'.format(cmd, self.target_node.private_ip_address), verbose=True)
+
```
Comment (roydahan) [inline sdcm/nemesis.py]: 1. Do we have more option of alter table that aren't intrusive for the load?
2. If we ALTER for adding a column during c-s write load, will it fail the c-s load?

File: sdcm/nemesis.py [hunk 4]
```diff
@@ -654,3 +666,17 @@ def disrupt(self):
...
+
+class ModifyTableCommentMonkey(Nemesis):
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.disrupt_modify_table_comment()
+
+
+class ModifyTableGCGraceSecondsMonkey(Nemesis):
```
Comment (roydahan) [inline sdcm/nemesis.py]: please remove the "Second" from the Monkey name.

File: sdcm/nemesis.py (hunk 3 above)
Comment (larisau) [inline sdcm/nemesis.py]: I used ALTER TABLE to change table properties only(as in the issue), bit not columns.
This type of ALTER does not affects c-s load.

File: sdcm/nemesis.py (hunk 4 above)
Comment (larisau) [inline sdcm/nemesis.py]: I can rename it to GraceTime

**Pull request merged**